File: csharp/uno/AddTwoNumbers/ListNode.cs
namespace Csharp.ListNode
{
    public class ListNode
    {
        public int val;
        public ListNode next;
        public ListNode(int x) { val = x; }
        public ListNode() { }
        public ListNode(int val = 0, ListNode next = null) { this.val = val; this.next = next; }
    }
}

File: Dart/unonew/lib/AddTwoNumbers/ListNode.dart
class ListNode {
  int val;
  ListNode? next;

  ListNode(this.val, [this.next]);
  ListNode([this.val = 0, this.next])

  @override
  String toString() {
    return 'ListNode{val: $val, next: $next}';
  }
}